htlcswitch/link.go

```diff
@@ -4643,3 +4578,82 @@ func (l *channelLink) resumeLink(ctx context.Context) error {
 
 	return err
 }
+
+// processRemoteUpdateAddHTLC takes an `UpdateAddHTLC` msg sent from the remote
```

Comment:
Nit: It would be really nice to describe a bit more what this function does in more detail.